homeassistant/components/eight_sleep/sensor.py

```diff
@@ -193,6 +206,14 @@ def device_state_attributes(self):
             # Skip attributes if sensor type doesn't support
             return None
 
+        if "fitness" in self._sensor_root:
+            state_attr = {ATTR_FIT_DATE: self._attr["date"]}
```

Comment:
Please declare and fill this dict all in one expression ✌
```python
state_attr = {
    ATTR_FIT_DATE: self._attr["date"],
    ATTR_FIT_DURATION_SCORE: self._attr["duration"],
    ...
}
```

Reply:
Corrected.  Thanks for the review.